config/database.yml

```diff
@@ -20,6 +20,8 @@ default: &default
   # For details on connection pooling, see Rails configuration guide
   # https://guides.rubyonrails.org/configuring.html#database-pooling
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
+  username: "postgres"
+  password: <%=1234%>
```

Comment:
Never use password and sensitive information directly. Google and discuss with your peers how credentials file work.